src/module_recipe.rs

```diff
@@ -136,6 +148,33 @@ impl<'a> Recipe<'a> {
             process::exit(1);
         })
     }
+
+    fn get_os_version(&self) -> Result<String> {
+        trace!("Recipe::get_os_version()");
+        check_command_exists("skopeo")?;
```

Comment:
What happens if this doesnt exist on the user system?

Reply:
Then it errors and the program exits, I could just default us to using the `image_version` defined in the recipe if we really want.

Reply:
Something to fallback on unless we force the CLI to  install the binary